main/ecrecover/FPSECP256K1/sqrtFpSecp256k1.zkasm

```diff
@@ -10,15 +10,19 @@
 ;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+VAR GLOBAL _C
```

Comment:
this could lead to confusion, since `C` is a register.
Maybe a more readable name is better

Reply:
Changed to _input